### PR TITLE
Explain non-LTS change wave versioning

### PR DESCRIPTION
**Pull request by BenVillalobos**

### CODE REVIEW

File: documentation/wiki/ChangeWaves-Dev.md [hunk 1]
```diff
@@ -12,6 +12,9 @@ The opt-out comes in the form of setting the environment variable `MSBuildDisabl
 ## Choosing a Change Wave for a New Feature
 This is determined on a case by case basis and should be discussed with the MSBuild team. A safe bet would be to check our [currently active Change Waves](ChangeWaves.md#change-waves-&-associated-features) and pick the version after the latest MSBuild version. This version corresponds to the latest version of Visual Studio.
 
+### Change Wave Versioning
+Change Wave features should match the LTS version of VS they were released with. Any feature requiring a changewave during a non-LTS release of VS should use the **NEXT** version number.
+
 # Developing With Change Waves in Mind
 For the purpose of providing an example, the rest of this document assumes we're developing a feature for MSBuild version **17.4**.
```
Comment (rainersigwald) [inline documentation/wiki/ChangeWaves-Dev.md:19]:
```suggestion
For the purpose of providing an example, the rest of this document assumes we're developing a feature for MSBuild version **17.3**.
```

?

**Review by rainersigwald — APPROVED** [review]

**Comment by Forgind** [comment]
Are we aligning with the nearest LTS release or just the even number equal to or one greater than the changewave? If, as a strawman, there were no LTS releases between 18.0 and 19.0, I'd still expect at least some change waves in 18.x for x > 0.

**Comment by Forgind** [comment]
Of note, 16.10 is _not_ LTS, whereas 16.11 is, yet we have a 16.10 wave and not a 16.11 wave. We should change one or the other. I'd vote to always use even-numbered change waves to make it a little easier to figure out what change wave you should use.

**Comment by Forgind** [comment]
Having just chatted with marcpopMSFT, it sounds like LTS versions are what we're going with going forward, and 16.10 was a mistake. I still slightly prefer going even to even over LTS to LTS, but it isn't a big deal.

File: documentation/wiki/ChangeWaves-Dev.md (hunk 1 above)
Comment (Forgind) [reply documentation/wiki/ChangeWaves-Dev.md:19]: It's fairly obvious how 17.4 should work, whereas 17.3 requires knowing whether it's attached to 17.2 or 17.4.

Comment (BenVillalobos) [reply documentation/wiki/ChangeWaves-Dev.md:19]: Fair, taking the commit.

Comment (rainersigwald) [reply documentation/wiki/ChangeWaves-Dev.md]: Yeah, I was wanting to exercise the "round up" path in the example.

**Pull request merged**